services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -293,16 +293,18 @@ export const DashboardPage = ({ reportType }: Props) => {
       </Box>
       {/* Only show SAR alert banner if the corresponding Work Plan is not approved */}
       <Box sx={sx.leadTextBox}>
-        {reportType === ReportType.SAR && !workPlanToCopyFrom && (
-          <Alert
-            title={sarVerbiage.alertBanner.title}
-            showIcon={true}
-            icon={alertIcon}
-            status={AlertTypes.ERROR}
-            description={sarVerbiage.alertBanner.body}
-            sx={sx.alertBanner}
-          />
-        )}
+        {reportType === ReportType.SAR &&
```

Comment:
This is just a nit, but when the conditional gets this long, I usually like to pull it into a variable, just to keep the template a bit cleaner.

Reply:
Sounds good, just updated this with a variable for the logic.